packages/manager/modules/pci/src/projects/project/ai/apps/components/update-scaling/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,22 @@
+{
+  "pci_app_update_app_scaling_title": "Modification du dimensionnement",
+  "pci_app_update_app_scaling_submit_label": "Confirmer",
+  "pci_app_update_app_scaling_cancel_label": "Annuler",
+  "pci_app_update_app_scaling_description": "Selon votre préférence, choisissez la stratégie de déploiement qui convient le mieux pour votre application : optez pour un dimensionnement fixe du nombre de réplicas sur lesquelles sera déployée votre application ou bien indiquez un minimum et maximum de réplicas pour vos déploiements afin d'en optimiser dynamiquement les performances et vous adapter au nombre de requêtes via l'option Autoscaling.",
```

Comment:
i noticed we use `app` in some translation and `application` here.
May be we could use `app`.

Reply:
Good point, I've changed the wording here: https://github.com/ovh/manager/pull/8001/commits/33a7214d2b0990fa80eb54768cc5b204824233b1